frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -199,6 +206,27 @@ export default class PivotTable extends Component {
       return null;
     }
 
+    const grid = this.bodyRef && findDOMNode(this.bodyRef);
+
+    // In cases where there are horizontal scrollbars are visible AND the data grid has to scroll vertically as well,
+    // the left sidebar and the main grid can get out of ScrollSync due to slightly differing heights
+    function needsScrollbarOffset() {
+      if (!grid) {
+        return false;
+      }
+      // check to see if the main data container has Y scrolling
+      const scrollsVertically = grid.scrollHeight > parseInt(grid.style.height);
+      const scrollsHorizontally = grid.scrollWidth > parseInt(grid.style.width);
+      // windows always has scrollbars, otherwise check for the Mac setting
+      const hasScrollbars =
+        window.navigator.platform === "Win32" || areScrollbarsVisible();
```

Comment:
Shouldn't `areScrollbarsVisible` always return true on Windows then? If not, maybe move the platform check into that function.

Reply:
Yup, goof on my part.